contracts/ChainlinkRegistry/ChainlinkRegistry.sol

```diff
@@ -19,8 +19,7 @@ contract ChainlinkRegistry is Governable, CollectableDust, IChainlinkRegistry {
   function assignFeeds(Feed[] calldata _feedsToAssign) external onlyGovernor {
     for (uint256 i = 0; i < _feedsToAssign.length; i++) {
       Feed memory _feed = _feedsToAssign[i];
-      if (address(_feed.base) == address(0) || address(_feed.quote) == address(0)) revert ZeroAddress();
```

Comment:
I think that these checks are not necessary. What do you think?

Reply:
Depends what the intent is. To check that we haven't screwed up, or Chainlink. If its to get more assurances from Chainlink, then I'd put it on, since they may f* up their process.

Reply:
We set the `base` and `quote`, not Chainlink, so it was only there to make sure we didn't make a mistake

But if we do make a mistake of setting one of this as the zero address:
- I don't think there will be any impact
- It can easily be fixed with another call

Reply:
Interesting... Ok! I'm not completely sold, but I get your point, let's just remove them then :)